torch/_inductor/codegen/common.py

```diff
@@ -438,14 +438,10 @@ def __init__(self, name, line):
         self.name = name
 
     def __call__(self):
-        # V.kernel may be null since this method may be called for the
-        # wrapper codegen where there is no specific kernel.
-        if (
-            self.name
-            not in (
-                V.graph.removed_buffers | getattr(V.kernel, "removed_buffers", set())
-            )
-            and self.name not in V.graph.inplaced_to_remove
+        if self.name not in (
+            V.graph.removed_buffers | V.kernel.removed_buffers
+        ) and self.name not in (
+            V.graph.inplaced_to_remove | V.kernel.inplaced_to_remove
```

Comment:
`self.name not in V.graph.removed_buffers and name not in V.kernel.removed_buffers` would be faster since it is O(1) rather than O(n).  There are two similar cases below.